src/app/features/manage-tokens/manage-tokens.tsx

```diff
@@ -0,0 +1,111 @@
+import { type ChangeEvent, useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { HStack, Stack } from 'leather-styles/jsx';
+
+import { Button } from '@leather-wallet/ui';
+
+import { BitcoinStandardsLoader } from '@app/features/asset-list/bitcoin/bitcoin-standards-loader/bitcoin-standards-loader';
+import { StacksAssetsLoader } from '@app/features/asset-list/stacks/stacks-assets-loader/stacks-assets-loader';
+import { useCurrentAccountIndex } from '@app/store/accounts/account';
+import { setToken, useEnabledTokens } from '@app/store/manage-tokens/manage-tokens.slice';
+import { Dialog } from '@app/ui/components/containers/dialog/dialog';
+import { Footer } from '@app/ui/components/containers/footers/footer';
+import { DialogHeader } from '@app/ui/components/containers/headers/dialog-header';
+import Toggle from '@app/ui/components/toggle/toggle';
+
+interface ManageTokensProps {
+  onClose(): void;
+}
+
+export type TokenProtocol = 'stacks' | 'bitcoin';
```

Comment:
This already exists as `SupportedBlockchains`, can import that instead.

Reply:
Can you update this @cbadawi ?